upstream-community-operators/percona/percona-xtradb-cluster-operator.0.3.0-rc1.clusterserviceversion.yaml

```diff
@@ -134,7 +147,7 @@ spec:
 
     ### Before You Start
 
-    1. Add the PXC user Secrets to Kubernetes. They should be placed in the
+    Add the PXC user Secrets to Kubernetes. They should be placed in the
```

Comment:
Removed this as it's only one step the user needs to do, plus it then is the same as the file in the community-operators directory.